base_name_search_improved/tests/test_name_search.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase, at_install, post_install
+
+
+@at_install(False)
+@post_install(True)
+class NameSearchCase(TransactionCase):
+
+    def setUp(self):
+        super(NameSearchCase, self).setUp()
+        phone_field = self.env.ref('base.field_res_partner_phone')
```

Comment:
Tests are failing because of this: the field XML Id changed to `base.field_res_partner__phone`